README.md

```diff
@@ -3,5 +3,11 @@ SymfonyMakerBundle
 
 Symfony Maker helps you creating empty commands, controllers, form classes,
 tests and more so you can forget about the required boilerplate code. This
-bundle replaces [SensioGeneratorBundle](https://github.com/sensiolabs/SensioGeneratorBundle)
-in modern Symfony applications.
+bundle replaces [SensioGeneratorBundle][1] in modern Symfony applications and
+requires using Symfony 3.4 or newer and [Symfony Flex][2].
+
+[Read the docs][3]
+
+[1]: https://github.com/sensiolabs/SensioGeneratorBundle
+[2]: https://symfony.com/doc/current/setup/flex.html
+[3]: Resources/doc/index.rst
```

Comment:
I would link to the rendered documentation on symfony.com instead. Reading reStructuredText files may not be much fun.

Reply:
Yes! We'll update this when/if docs are published on symfony.com